spec/models/collection_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe Collection, :type => :model do
   before do
     @user = FactoryGirl.create(:user)
-    @collection = Collection.new(title: "test collection").tap do |c|
+    @collection = Collection.new(id: 'mock-collection-with-members', title: "test collection") do |c|
```

Comment:
Do you need to specify the ID here for some reason?  You don't seem to be using it later in the test, so is it useful?